.github/workflows/pull-request.yml

```diff
@@ -12,7 +12,7 @@ on:
         description: 'devel'
         required: false
         type: string
-        default: false
+        default: 'false'
```

Comment:
is conversion to string needed here?

Reply:
it was making the checker failing and as you see it's type string it make sense to have it a default value string